kolibri/core/exams/single_user_assignment_utils.py

```diff
@@ -0,0 +1,106 @@
+from .models import ExamAssignment
```

Comment:
This file is virtually identical to kolibri/core/lessons/single_user_assignment_utils.py -- in terms of the overall structure -- some field names and details are different.

I'd be happy to try to DRY it up, it just might make it a bit more "magical" (e.g. a function containing this logic that gets passed models and field names etc, and then the comments in here etc would need to be generalized as well -- might hurt readability.

Reply:
agree with the copy/paste strategy for code maintainability